leaderelection/dummy_manager.go

```diff
@@ -0,0 +1,35 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package leaderelection
+
+import "context"
+
+// A dummy leader-election manager to implement the ElectionManager interface.
+// Use to keep the leaderelection code clean and without a non k8s logic.
+
+type DummyManager struct{}
```

Comment:
Let's find a better name for that.
Maybe not even using k8s terminology.
The interface should be named `UniquenessManager`
And then, the k8s implementation `LeaderelectionManager`
And our new one, `DefaultUniqueManager`.
WDYT?

Reply:
Sounds good.
New pkg name  - `uniqueness`